### ISSUE TITLE
hv_storvsc_on_iocompletion in azure

### ISSUE THREAD

**Issue by eyberg** [nanovms/nanos#1235]
```
➜  atest ./logs-local.sh
 VMBUS CHAN: offer ch_id 2 ch_subidx 0

 VMBUS CHAN: chan2 subidx0 offer

 VMBUS CHAN: chan2 assigned to cpu0 [vcpu0]

 VMBUS CHAN: offer ch_id 3 ch_subidx 0

 VMBUS CHAN: chan3 subidx0 offer

 VMBUS CHAN: chan3 assigned to cpu0 [vcpu0]

 VMBUS CHAN: offer ch_id 15 ch_subidx 0

 VMBUS CHAN: chan15 subidx0 offer

 VMBUS CHAN: chan15 assigned to cpu0 [vcpu0]

 VMBUS CHAN: offer ch_id 16 ch_subidx 0

 VMBUS CHAN: chan16 subidx0 offer

 VMBUS CHAN: chan16 assigned to cpu0 [vcpu0]

 VMBUS CHAN: offer ch_id 17 ch_subidx 0

 VMBUS CHAN: chan17 subidx0 offer

 VMBUS CHAN: chan17 assigned to cpu0 [vcpu0]

 VMBUS CHAN: OPEN_BR, cbarg = 1600c00

 VMBUS CHAN: chan2 update cpu0 flag_cnt to 1

 VMBUS CHAN: gpadl_conn(chan2) succeeded

 VMBUS CHAN: chan2 opened

 VMBUS CHAN: OPEN_BR, cbarg = 1601000

 VMBUS CHAN: gpadl_conn(chan3) succeeded

 VMBUS CHAN: chan3 opened

ffffffff80064b51        (vmbus_chan_task + 0000000000000041/0000000000000077)
ffffffff80065c62        (vmbus_event_proc + 00000000000000d2/00000000000001a2)
ffffffff8006658f        (vmbus_chan_poll_messages + 000000000000001f/0000000000000031)
ffffffff8006b168        (storvsc_probe + 0000000000000218/0000000000000704)
ffffffff80062eca        (hyperv_probe_devices + 00000000000000aa/0000000000000113)
ffffffff80077671        (init_service_new_stack + 0000000000000281/0000000000000303)
assertion vm_srb->sense_info_len <= request->sense_info_len failed in /home/eyberg/go/src/github.com/nanovms/nanos/src/hyperv/storvsc/storvsc.c: hv_storvsc_on_iocompletion() on line 554; halt
```

**Comment by eyberg** [comment]
I think you can close this @mkhon ?

**Comment by mkhon** [comment]
Fixed by this commit: 2dc264864dc7e761740ceffbfaaab2241bab0b6b
